lib/gpu/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "gpu"
+version = "0.0.1"
+authors = [
+    "Andrey Vasnetsov <vasnetsov93@gmail.com>",
+    "Ivan Pleshkov <pleshkov.ivan@gmail.com>",
+    "Qdrant Team <info@qdrant.tech>",
+]
+license = "Apache-2.0"
+edition = "2021"
+
+[features]
+gpu = [
```

Comment:
I decided to include crate anyway but if `gpu` feature is disabled, all codes and dependencies are not presented, crate will be empty. It allows to skip GPU for regular build and allow to use cargo chef without errors (cargo prepare wants to include this crate even if feature is disabled)